src/main/java/lanat/helpRepresentation/ArgumentRepr.java

```diff
@@ -47,7 +62,7 @@ private ArgumentRepr() {}
 		return ArgumentRepr.getRepresentation(arg) + ":\n" + HelpFormatter.indent(desc, arg);
 	}
 
-	static String getDescriptions(@NotNull List<@NotNull Argument<?, ?>> arguments) {
+	static String getDescriptions(final @NotNull List<@NotNull Argument<?, ?>> arguments) {
```

Comment:
Is the final modifier really needed?
```suggestion
	static String getDescriptions(@NotNull List<@NotNull Argument<?, ?>> arguments) {
```